### DISCUSSION TITLE
DisabledMenus in igconfig.admin.json?

### DISCUSSION THREAD

**Discussion by VDI-Bob** [d2phap/ImageGlass#1947] [Q&A] [answered]
Hello, I would like to hide some points in the menu ( Layout, Tools, Settings, Help). I have found two points for this, but I don't understand how to combine them. 

1. in the igconfig.json there is "DisabledMenus": [], 
I would like to transfer these to an igconfig.admin.json.
2. a list of the menu items and a line of code <Item key="DisabledMenus" value="mnuMainSetAsDesktop;mnuMainSetAsLockImage" /> https://github.com/d2phap/ImageGlass/issues/1498#issuecomment-1482911514

Question: How do I combine these two items in a json to be able to hide settings?

Thank you for the support

**Comment by d2phap** [comment, marked as answer]
You can create `igconfig.admin.json` file with only the `DisabledMenus` setting:
```json
{
  "_Metadata": {
    "Description": "ImageGlass configuration file",
    "Version": 9.1
  },
  "DisabledMenus": [
    .............
  ]
}
```

For ImageGlass 9, you should use the menu names (start with `Mnu` prefix) here. Although I will try not to change the menu names as well as the [ImageGlass API methods (begins with `IG_`)](https://github.com/d2phap/ImageGlass/blob/develop/Source/ImageGlass/FrmMain/FrmMain.IGMethods.cs), you can find the updated menu names in [`FrmMain.Designer.cs`](https://github.com/d2phap/ImageGlass/blob/develop/Source/ImageGlass/FrmMain.Designer.cs) file.
https://github.com/d2phap/ImageGlass/blob/b11e4e4c6f256f24210f4021249417124dbdcabb/Source/ImageGlass/FrmMain.Designer.cs#L1229-L1361


This is how it looks like when using the `DisabledMenus`: the disabled menu item has a lock icon next to the text
![image](https://github.com/d2phap/ImageGlass/assets/3154213/ed165963-e3bd-4e94-803a-81d0bd3dc2a6)

**Comment by VDI-Bob** [comment]
Thank you for the quick response. It works very well!
